documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,257 @@
+
+NOTE: In this example, JSON converters are specified. The converters you specify must be in a data format supported by Kafka.
+
+A distributed Kafka Connect cluster has a group ID and a set of internal connector topics.
 Kafka Connect instances are configured by default with the same:
 
 * Group ID for the Kafka Connect cluster
 * Kafka topic to store the connector offsets
 * Kafka topic to store connector and task status configurations
 * Kafka topic to store connector and task status updates
 
-If multiple different Kafka Connect instances are used, these settings must reflect each instance.
+If multiple different Kafka Connect instances are used, these settings must be unique for the workers of each Kafka Connect cluster created.
 
-[discrete]
-== Example YAML showing Kafka Connect configuration
+.Worker configuration that must be unique for each Kafka Connect instance
 [source,yaml,subs="attributes+"]
 ----
 apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnect
-metadata:
-  name: my-connect
+# ...
 spec:
-  # ...
   config:
     group.id: my-connect-cluster
     offset.storage.topic: my-connect-cluster-offsets
     config.storage.topic: my-connect-cluster-configs
     status.storage.topic: my-connect-cluster-status
-  # ...
 ----
 
-[discrete]
-== Connectors
+The group ID identifies the Kafka Connect worker nodes. Kafka Connect clusters cannot share the ID or topics as it will create errors.
+The names of the connectors used by each Kafka Connect instance must also be unique.
 
-Connectors are configured separately from Kafka Connect.
-The configuration describes the source input data and target output data to feed into and out of Kafka Connect.
-The external source data must reference specific topics that will store the messages.
+You can also link:{BookURLDeploying}#creating-new-image-using-kafka-connect-build-str[deploy connector plugins^] using the `build` configuration of the `KafkaConnect` resource.
+Connector plugins provide specific configuration for the creation of connector instances.
+
+After connector plugins have been added to the Kafka Connect deployment, you can use Strimzi’s `KafkaConnector` custom resource or the Kafka Connect API to manage connector instances.
+You can also create new connector instances using these options.
 
-Kafka provides two built-in connectors:
+The Cluster Operator manages Kafka Connect clusters deployed using the `KafkaConnect` resource and connectors created using the `KafkaConnector` resource.
 
---
-* `FileStreamSourceConnector` streams data from an external system to Kafka, reading lines from an input source and sending each line to a Kafka topic.
-* `FileStreamSinkConnector` streams data from Kafka to an external system, reading messages from a Kafka topic and creating a line for each in an output file.
---
+[discrete]
+== Connector plugin configuration
+
+Connector plugins provide the configuration for connecting to an external data source.
```

Comment:
Again, I find the word "configuration" here confusing. I don't think of a (set of) jar file(s) (which is what a plugin _is_) as "configuration". I think most people think of configuration as either some kind of text file or similar with a bunch of settings, or a UI with a bunch of settings. 

I would prefer if we use the word "configuration" to refer to the `config` part of the resources concerned, and the word "resource" for the CRs and avoid conflating either with "plugin".

Reply:
Changed to:

> Connector plugins provide the implementation for connecting to specific types of external data system.
> Plugins contain a set of files that define the implementation.